cluster/examples/kubernetes/ceph/cluster-on-pvc.yaml

```diff
@@ -23,3 +31,45 @@ spec:
         resources:
           requests:
             storage: 10Gi
+  cephVersion:
+    image: ceph/ceph:v14.2.1-20190430
+    allowUnsupported: false
+  dashboard:
+    enabled: true
+  network:
+    hostNetwork: false
+  storage:
+    storageClassDeviceSets:
+    - name: cluster1-set1
```

Comment:
For the example, let's name it `set1`. Users can get fancy with longer names if they want

Reply:
updated